iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -18,6 +18,11 @@ import { escapeRegex } from './util.js'
 import { IXNode } from './ixnode.js';
 import { setDefault } from './util.js';
 import { DocOrderIndex } from './docOrderIndex.js';
+import { MessageBox } from './messagebox.js';
+
+const CONTINUATION_LIMIT = 10000;
```

Comment:
Yes, we could include something in the report data block, although I think I'd prefer to add it as a "don't show document size warning" flag, rather than making the limit configurable.  I think it's likely that we'll want to add other size checks, and making them all independently configurable at a document-level seems unnecessary.

In any event, I should probably move this limit to the iXBRLViewer options object, so that it can be overridden in extensions.

Reply:
Awesome, all of that seems good to me